### PR TITLE
fix(eventHandler): fix changeHandler for stop

### PR DESCRIPTION
**Pull request by younkue**
## Issue

#169

### CODE REVIEW

File: src/eventHandler.js [hunk 1]
```diff
@@ -93,7 +93,20 @@ export default superclass => class extends superclass {
...
+		} else {
+			e.stop();
+			// release, animationEnd
+			touch.distance = 0;
+			touch.direction = direction || touch.direction;
+			touch.destPos = 0;
+			this._adjustContainerCss("end");
+			touch.holding = false;
+			this._setPointerEvents();
+			conf.touch.isTrusted = false;
```
Comment (WoodNeck) [inline src/eventHandler.js]: I think you can use just `touch` here.

File: src/eventHandler.js [hunk 2]
```diff
@@ -93,7 +93,20 @@ export default superclass => class extends superclass {
...
 		);
 
-		(eventRes || eventRes === null) && this._setTranslate([-pos, 0]);
+		if (eventRes || eventRes === null) {
+			this._setTranslate([-pos, 0]);
+		} else {
+			e.stop();
+			// release, animationEnd
+			touch.distance = 0;
+			touch.direction = direction || touch.direction;
```
Comment (WoodNeck) [inline src/eventHandler.js]: Why we should maintain direction here? Isn't it like reset everything kind of phase?

File: src/eventHandler.js [hunk 3]
```diff
@@ -93,7 +93,20 @@ export default superclass => class extends superclass {
...
+			e.stop();
+			// release, animationEnd
+			touch.distance = 0;
+			touch.direction = null;
+			touch.destPos = 0;
+			this._adjustContainerCss("end");
+			touch.holding = false;
+			this._setPointerEvents();
+			touch.isTrusted = false;
+			this._setPhaseValue("end");
```
Comment (jongmoon) [inline src/eventHandler.js]: @younkue How about grouping code into 2 groups. One group for `touch`, the other group is to call `method`?

Comment (jongmoon) [reply src/eventHandler.js]: @younkue Are these methods affected by order?

**Pull request merged**